docs/index.html

```diff
@@ -31,128 +26,90 @@
+						</p>
 					</div>
 				</div>
 
-				<div class="section-light">
-					<div class="section-inner">
-						<h2>How ylem works</h2>
-						<p><b>yelm</b> is "observing" your state and will automatically re-render your components when anything changes - you should never call <span class="code-highlight-light">setState</span>. More importantly, <b>ylem</b> only re-renders your components when something <i>actually</i> changes. This means you never have to implement <span class="code-highlight-light">shouldComponentUpdate</span> or use <span class="code-highlight-light">PureComponent</span>. Humans can never implement <span class="code-highlight-light">shouldComponentUpdate</span> across an entire app as efficiently as a code solution like <b>ylem</b>. This alone will save your team hours of time while increasing your apps consistency and rendering efficiency (for free).</p>
-					</div>
+				<div class="triptych">
+					<a href="#easy-for-developers">
+						<div class="large">Easy for developers</div>
+					</a>
+					<a href="#removes-boilerplate">
+						<div class="large">Removes boilerplate</div>
+					</a>
+					<a href="#faster-rendering">
+						<div class="large">Faster rendering</div>
+					</a>
 				</div>
 
-				<div class="section-2 section-inline">
+				<div class="content-section">
 					<div class="section-inner">
 						<hgroup>
-							<h2>Easy for developers of all skill levels to understand.</h2>
+							<h2><a name="easy-for-developers">Easy for developers of all skill levels to understand.</a></h2>
 							<h3>If you know <span class="highlight">JavaScript</span>, you already know <span class="highlight">ylem</span>.</h3>
 						</hgroup>
 
 						<div class="code-container">
 							<div class="code-container_left">
 								<div class="code-container_title">
 									<h3>React</h3>
+									<p>Call <code>setState</code> with an updated copy of the old state</p>
 								</div>
 								<pre><code class="language-js">
-								import React, { Component } from 'react';
-
-								class Clock extends Component {
-								  constructor(props) {
-								    super(props);
-								    this.state = { date: new Date() };
-								  }
-
-								  componentDidMount() {
-								    this.timerID = setInterval(() => {
-								      this.setState({ date: new Date() });
-								    }, 1000);
-								  }
-
-								  componentWillUnmount() {
-								    clearInterval(this.timerID);
-								  }
-
-								  render() {
-								    const { date } = this.state;
-								    return (
-								      &lt;h2&gt;
-								        It is {date.toLocaleTimeString()}.
-								      &lt;/h2&gt;
-								    );
-								  }
-								}
+								// Add new todo
+								this.setState({
+								  todos: this.state.todos.concat({
+								    title: 'Dishes',
+								    complete: false
+								  })
+								});
+
+								// Set the first todo as complete
+								const clone = this.state.todos.slice();
+								clone[0].complete = true;
```

Comment:
*actually*, you're still modifying state here. to do it properly, you'd need this (also using spread cause its better =P):
```
const todos = [ ...todos ];
todos[0] = {
	...todos[0],
	complete: true,
};

this.setState({ todos });
```

Reply:
I was avoiding ...spread to make this more palatable for the masses, but you are correct about muting the existing object. I will fix that.